examples/topic-metadata.rs

```diff
@@ -36,7 +36,10 @@ struct Offsets {
 
```

Comment:
after updating my rust set up, I realized this small program gives the following compilation warning:

```
warning: unused `#[macro_use]` import, #[warn(unused_imports)] on by default
 --> examples/topic-metadata.rs:4:1
  |
4 | #[macro_use]
  | ^^^^^^^^^^^^
```

may i ask you to remove the unused macro item here as part of this PR?